UnitySDK/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -284,6 +277,7 @@ private void InitializeEnvironment()
             catch
             {
                 communicator = null;
+#if UNITY_EDITOR
```

Comment:
Can you update the comments above to include the change in logic?

Reply:
Updated comment